scss/_common.scss

```
@@ -0,0 +1,87 @@
+.button {
+  font-family: "Aleo", serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;
+  letter-spacing: 0.07em;
+  text-align: center;
+  height: 60px;
+  width: 213px;
```

думаю тоже лучше паддинг и `min-width`, вдруг нужно будет больше текста в кнопке
```suggestion
  padding: 20px 28px;
  min-width: 213px;
```